server/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.13.7-buster AS GO_SERVER
+
+WORKDIR /
+
+RUN apt install git && git clone https://github.com/keycloud/keycloud.git && cd keycloud && git checkout KC-82 && cd server && CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o server .
```

Comment:
As we use the Dockerfile inside the github repo itself, it would be better to copy the files from the host instead of pulling it again inside the image. This also automatically triggers an update of the whole build when rebuild with updated files on the host machine.

Reply:
This way, we could also stay with the old .ini file configuration as you can simply edit it on the host machine. But i also prefer the solution with enviroment variables